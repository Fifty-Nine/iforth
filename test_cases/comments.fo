."test (not a) comment\n"
(but every thing here is definitely a comment ."bad")
(this is a comment) ."not a comment\n" (also a comment)
