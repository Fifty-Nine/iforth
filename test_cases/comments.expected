test (not a) comment
not a comment
